Documentation/CRDs/Object-Storage/ceph-object-store-crd.md

```diff
@@ -139,6 +139,16 @@ All the other settings from the gateway section will be ignored, except for `sec
 The [zone](../../Storage-Configuration/Object-Storage-RGW/ceph-object-multisite.md) settings allow the object store to join custom created [ceph-object-zone](ceph-object-multisite-crd.md).
 
 * `name`: the name of the ceph-object-zone the object store will be in.
+* `endpoint`: the zone endpoint associated with the object-store. See description below.
```

Comment:
Why don't we just tell users how to set this value here? Were I a user, I think I would find the description below more confusing than helpful.

Suggest instead:
> If this zone cannot be accessed from other peer Ceph clusters via the ClusterIP Service 
endpoint created by Rook, you must set this to the externally reachable endpoint. You must also
include the port in the definition. For example: "https://my-object-store.my-domain.net:443".
In many cases, you should set this to the endpoint of the ingress resource that makes the
CephObjectStore associated with this CephObjectStoreZone reachable to peer clusters.

Reply:
I almost suggested to combine to the single paragraph as well, I think that sounds better too

Reply:
Sounds good. I will revise this section to include the suggested description.